pandas/io/clipboard.py

```diff
@@ -83,8 +83,12 @@ def to_clipboard(obj, excel=None, sep=None, **kwargs):  # pragma: no cover
             if sep is None:
                 sep = '\t'
             buf = StringIO()
-            obj.to_csv(buf, sep=sep, **kwargs)
-            clipboard_set(buf.getvalue())
+            # clipboard_set (pyperclip) expects unicode
+            obj.to_csv(buf, sep=sep, encoding='utf-8', **kwargs)
```

Comment:
so ``encoding`` is a valid kwarg. So instead put this in ``to_clipboard(....., encoding=None, **kwargs)``.

Then check for encoding, if its not ``None`` or ``utf-8`` I would raise an error, then pass onto pyperclip. add a test case for this.